File: RealEstate-Source-Code-v6.0/AdminPanel-NetCore/Entities/Property.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RealEstate.Entities
{
    public partial class Property
    {
        [Key]
        public int Id { get; set; }
        public int PropertyTypeId { get; set; }
        public int PropertyCategoryId { get; set; }
        public int UserId { get; set; }
        public int CityId { get; set; }
        public bool Featured { get; set; }
        public bool Visible { get; set; }
        [Required]
        [StringLength(250)]
        public string Title { get; set; }
        [Required]
        public string Description { get; set; }
        [Required]
        public string ImageNames { get; set; }
        public int Size { get; set; }
        public int BedRoomCount { get; set; }
        public int BathRoomCount { get; set; }
        public int KitchenRoomCount { get; set; }
        public int ParkingCount { get; set; }
        [Required]
        public string AdditionalFeatures { get; set; }
        [Column(TypeName = "numeric(18, 2)")]
        public decimal Price { get; set; }
        [Required]
        [StringLength(10)]
        public string Currency { get; set; }
        [Required]
        public string Address { get; set; }
        [Column(TypeName = "decimal(9, 6)")]
        public decimal Latitude { get; set; }
        [Column(TypeName = "decimal(9, 6)")]
        public decimal Longitude { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime CreatedDate { get; set; }

        [ForeignKey(nameof(CityId))]
        [InverseProperty("Property")]
        public virtual City City { get; set; }
        [ForeignKey(nameof(PropertyCategoryId))]
        [InverseProperty("Property")]
        public virtual PropertyCategory PropertyCategory { get; set; }
        [ForeignKey(nameof(PropertyTypeId))]
        [InverseProperty("Property")]
        public virtual PropertyType PropertyType { get; set; }
        [ForeignKey(nameof(UserId))]
        [InverseProperty("Property")]
        public virtual User User { get; set; }
    }
}

File: RealEstate-Source-Code-v6.0/AdminPanel-Laravel10/app/Models/Property.php
<?php

namespace App\Models;

use App\Models\User;
use Illuminate\Database\Eloquent\Model;

class Property extends Model
{
    protected $table = "properties";
    protected $fillable = [
        'property_type_id',
        'property_category_id',
        'user_id',
        'city_id',
        'featured',
        'title',
        'description',
        'image_names',
        'size',
        'bed_room_count',
        'bath_room_count',
        'kitchen_room_count',
        'parking_count',
        'additional_features',
        'price',
        'currency',
        'address',
        'latitude',
        'longitude',
        'visible',
    ];

    public function property_type()
    {
        return $this->belongsTo(PropertyType::class, 'property_type_id');
    }

    public function property_category()
    {
        return $this->belongsTo(PropertyCategory::class, 'property_category_id');
    }

    public function user()
    {
        return $this->belongsTo(User::class, 'user_id');
    }

    public function city()
    {
        return $this->belongsTo(City::class, 'city_id');
    }
}